finish/src/main/liberty/config/server.xml

```diff
@@ -4,9 +4,7 @@
     <feature>jaxrs-2.0</feature>
     <feature>jsonp-1.0</feature>
     <feature>cdi-1.2</feature>
-    <!-- tag::config[] -->
-    <feature>mpConfig-1.2</feature>
-    <!-- end::config[] -->
```

Comment:
I already remove the extra config feature last time, so please don't remove this one.